packages/devtools_app/lib/src/inspector/inspector_tree.dart

```diff
@@ -204,6 +204,13 @@ abstract class InspectorTreeNode {
     }
     final String name = diagnostic.name;
     TextStyle textStyle = textStyleForLevel(diagnostic.level);
+    if (diagnostic.isError) {
+      if (diagnostic.distanceFromError <= 0) {
+        textStyle = TextStyle(color: Colors.red, fontWeight: FontWeight.bold);
```

Comment:
lets move these TextStyle objects to the file that has the textStyleForLevel constants.

We'll also need to make sure the colors work ok in light and dark mode. I believe adding &theme=dark at the end of the URL will trigger dark mode.
Dark mode is the default when launching devtools from VSCode.